pkg/tbtc/maintainer/chain.go

```diff
@@ -2,12 +2,38 @@ package maintainer
 
 import (
 	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/chain"
 )
 
 // BitcoinDifficultyChain is an interface that provides the ability to
 // communicate with the Bitcoin difficulty on-chain contract.
 type BitcoinDifficultyChain interface {
```

Comment:
Shouldn't this interface be named after the contract as we do with other contracts?
```suggestion
type LightRelayChain interface {
```

Reply:
@nkuba 
There are different ideas how to name this interface.
@pdyraga suggested naming it `BitcoinDifficultyChain` and avoid using `Relay` in the name to avoid confusion with other parts of our system.

Reply:
Yep, I'd also vote to leave it as is. We don't need to have 1:1 mappings between contract names and chain interfaces. Contract names are just implementation details that can change over time. Also +1 for not using the "relay" word as it's a constant source of confusion.